packages/wrangler/src/config/config.ts

```diff
@@ -30,6 +30,8 @@ export type RawConfig = Partial<ConfigFields<RawDevConfig>> &
 export interface ConfigFields<Dev extends RawDevConfig> {
 	configPath: string | undefined;
 
+	worker_metadata: string | undefined;
```

Comment:
I feel like this setting should be part of each worker configuration, right now this puts in the global scope right?

Reply:
This is where the top level config live, not sure what you mean by "global scope" it doesnt affect the ambient types like `Env`